zeppelin-web-angular/projects/zeppelin-sdk/.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+  "extends": "../../.eslintrc.json",
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts"],
+      "parserOptions": {
+        "project": ["projects/zeppelin-sdk/tsconfig.lib.json", "projects/zeppelin-sdk/tsconfig.spec.json"],
```

Comment:
```suggestion
        "project": ["zeppelin-web-angular/projects/zeppelin-sdk/tsconfig.lib.json", "zeppelin-web-angular/projects/zeppelin-sdk/tsconfig.spec.json"],
```

<img width="637" height="67" alt="Image" src="https://github.com/user-attachments/assets/be9c53f0-41e6-45eb-a956-a1f53004e411" />

Reply:
I added path mapping in `e2e/tsconfig.json` and it seems to work.